etl/steps/data/meadow/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,49 @@
+"""Load a snapshot and create a meadow dataset."""
```

Comment:
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)